besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -779,6 +780,12 @@ static class JsonRPCHttpOptionGroup {
         paramLabel = MANDATORY_LONG_FORMAT_HELP,
         description = "Specifies the maximum request content length. (default: ${DEFAULT-VALUE})")
     private final Long rpcHttpMaxRequestContentLength = DEFAULT_MAX_REQUEST_CONTENT_LENGTH;
+
+    @Option(
+        names = {"--pretty-json-enabled"},
```

Comment:
```suggestion
        names = {"--json-pretty-print-enabled"},
```

Reply:
json should definitely come first. and I think "pretty print" rather than just "pretty"